viz.yaml

```diff
@@ -28,6 +28,12 @@ info:
     sp:
       repo: CRAN
       version: 1.2.7
+    sbtools:
+      repo: CRAN
+      version: 1.1.6
+    geojsonio:
+      repo: CRAN
+      version: 0.6.0
```

Comment:
It'd be nice to think about removing all of these declared package dependencies as we switch to Docker (because now the Dockerfile is where the decisions actually get made about which package version gets used). Is now too soon to do so, and if not, is it as simple as deleting all the lines in this subsection, or does vizlab give errors if you do that?

Reply:
We just would need to remove the `checkVizPackages()` call in`vizmake()`, since that requires there to be a packages yaml block.  Otherwise, I think we could probably just drop the packages section, unless we don't feel confident relying on the package versions that come through the rocker images.   

There is probably a fair bit of code associated with `checkVizPackages` that could either be removed from `vizlab`, unless we wanted to preserve a way of checking package versions against some reference.

Reply:
OK, thanks for thinking it through. I'll create a GitHub issue in vizlab for this, and we can resolve it (including cleaning up the extra code) after we've had a chance to kick the tires on the Docker option for a while.

What happens if you make a packages YAML block but leave it empty?

Reply:
Added this issue: https://github.com/USGS-VIZLAB/vizlab/issues/395

Reply:
That just returns a null from `checkVizPackages`, I think that would work